src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-app2-overseas-about-to-submit.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
is all this data needed for this specific callback? might be easier to manage if you just restrict it to the data used in the callback

Reply:
im not sure as I copied it from a similar test with some changes to the request I made.